agera/src/test/java/com/google/android/agera/ObservablesTest.java

```diff
@@ -325,6 +325,32 @@ public void shouldUpdatePerMillisecondObservable() {
   }
 
   @Test
+  public void shouldUpdateCompositeOfPerMillisecondObservable() {
+    final long expectedDelayedTime = scheduler.getCurrentTime() + FILTER_TIME;
+    updatable.addToObservable(compositeObservable(perMillisecondObservable(
+        FILTER_TIME, updateDispatcher), updateDispatcher()));
+
+    updateDispatcher.update();
+    idleMainLooper(FILTER_TIME);
+
+    assertThat(updatable, wasUpdated());
+    assertThat(scheduler.getCurrentTime(), greaterThanOrEqualTo(expectedDelayedTime));
```

Comment:
it check that idleMainLooper(FILTER_TIME); works, doesn't it? why do you need it here?